canonical/ubuntu-advantage-client
# ros: unhide ros entitlements

## Notes

There is movement on the ticket to update the ros infrastructure. We need to be ready to test on staging and give the go ahead for moving to production and then release this asap.


**Note that now that the ros tests are using the production contract server, we need to set UACLIENT_BEHAVE_CONTRACT_TOKEN to a token that entitles esm-apps and ros. I am updating jenkins to use such a token.**

## Proposed Commit Message


```
ros: unhide ros entitlements
```

## Test Steps

Enable ros: verify it auto enables esm infra and apps

enable ros-updates: verify it auto enables esm infra and apps and ros

## Desired commit type::


## Checklist:
